jetty-server/src/test/java/org/eclipse/jetty/server/RequestTest.java

```diff
@@ -639,6 +639,56 @@ public boolean check(HttpServletRequest request, HttpServletResponse response)
         assertThat(responses, startsWith("HTTP/1.1 200"));
     }
 
+    @Test
+    public void testContentLength_ExceedsMaxInteger() throws Exception
```

Comment:
not a fan of these _ in method names!

Reply:
```
[jetty.project-9.4.x]$ $ find . -wholename "*/src/test/java/*.java" -exec grep "void test[A-Za-z0-9]*_[A-Za-z0-9]" {} \; | wc -l
934
```

We have 934 instances of this in our codebase.